include/apps/tools/OswAppCalculator.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <osw_hal.h>
+#include <osw_ui.h>
+#include "osw_app.h"
+
+
+class OswAppCalculator : public OswApp {
+  public:
+    OswAppCalculator(void) {
+        ui = OswUI::getInstance();
+    };
+    virtual void setup() override;
+    virtual void loop() override;
+    virtual void stop() override;
+    void draw();
```

Comment:
I'd appreciate it if you could describe and comment on how functions work.
Use it. `@param` and `/**/`.